cartesi-rollups/development/community-tools.md

```diff
@@ -0,0 +1,179 @@
+---
+id: community-tools
+title: Community tools
+---
+
+Several tools created and maintained by the community streamline the dApp creation process on Cartesi Rollups.
+
+
+## RISC-V Verifiable Entertainment System (RIVES)
+
+A low-level verifiable fantasy console for building small games on a special RISC-V Linux machine.
+
+- **Features**:
+	- Create small games using a minimal API and simple tools.
+	- Use any programming language with support for RISC-V.
+	- Play games on any device with WebAssembly support.
+	- Record game sessions into tiny tapes for replay and verification.
+	- Custom leaderboards for each cartridge.
+
+- **Resources**:
+	- [RIVES Documentation](https://rives.io/docs/category/riv)
+	- [RIVES Playground](https://rives.io/playground)
+
```

Comment:
Wouldn`t be nice here to also add app.rives.io ? Since it is also nice to see the rives working.